configure

```diff
@@ -786,6 +787,7 @@ infodir
 docdir
 oldincludedir
 includedir
+runstatedir
```

Comment:
What's the purpose/need for the new directory?

Reply:
That is a good question. We re-ran `autoconf`and I think it generated that into the configure script. I can remove that.